lib/message.rb

```diff
@@ -0,0 +1,34 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class Message
```

Comment:
I like how you organised the class - it makes it easy to understand what's going on! In terms of testing (as I saw you mentioned it in the readme), you could have used some of the info generated by the message itself to check if it was sent or not

Reply:
Yep I agree with Alina